pkgs/development/python-modules/pylibftdi/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, pythonAtLeast
+, httpserver
+, libftdi1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  pythonAtLeast = "3"; # compatibility with httpserver
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "132c44rs0f2zm22hnhg93p47aaw6llpfzavc64dcqrn8719mlqfw";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    pkgs.libusb1
+  ];
+
+  doCheck = false;
```

Comment:
can you add a comment as to why tests are not running?

Reply:
I've added several patches that make the tests pass.

Reply:
tests are really important to verify the package is still valid. We have the ability to largely disregard version bounds, and tests are kind of our saving grace.